From: Sasha Neftin <sasha.neftin@intel.com>
Date: Sun, 23 Jun 2019 13:42:32 +0300
Subject: igc: Remove the unused field from a device specification structure
Patch-mainline: v5.4-rc1
Git-commit: 6145787d5e2eb973fa9f73d228d7a6a13650b1d1
References: jsc#SLE-7966

This patch comes to clean up the device specification structure.

Signed-off-by: Sasha Neftin <sasha.neftin@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/intel/igc/igc_hw.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/net/ethernet/intel/igc/igc_hw.h
+++ b/drivers/net/ethernet/intel/igc/igc_hw.h
@@ -184,12 +184,7 @@ struct igc_fc_info {
 };
 
 struct igc_dev_spec_base {
-	bool global_device_reset;
-	bool eee_disable;
 	bool clear_semaphore_once;
-	bool module_plugged;
-	u8 media_port;
-	bool mas_capable;
 };
 
 struct igc_hw {
